HW31/task2.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+
+def get_coordinates(city):
+    response = requests.get(f"https://geocoding-api.open-meteo.com/v1/search?name={city}")
+    data = response.json()
+    if "results" in data:
+        result = data["results"][0]
+        latitude = result["latitude"]
+        longitude = result["longitude"]
+        return latitude, longitude
+    return None
+
+
+city = input("Please enter city name: ")
+coordinates = get_coordinates(city)
+
+if coordinates:
+    latitude, longitude = coordinates
+    res = requests.get(f"https://api.open-meteo.com/v1/forecast?latitude={latitude}&longitude={longitude}&hourly=temperature_2m,relativehumidity_2m,windspeed_10m")
```

Comment:
nit: better to use `params` for request

Reply:
I don't understand what you mean, can you elaborate?

Reply:
https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls